pkg/protocol/syncmanager/trivialsyncmanager/syncmanager.go

```diff
@@ -96,20 +96,20 @@ func (s *SyncManager) SyncStatus() *syncmanager.SyncStatus {
 	s.lastConfirmedBlockSlotLock.RLock()
 	s.latestCommitmentLock.RLock()
 	s.latestFinalizedSlotLock.RLock()
-	s.lastPrunedSlotLock.RLock()
+	s.lastPrunedEpochLock.RLock()
 	defer s.lastAcceptedBlockSlotLock.RUnlock()
 	defer s.lastConfirmedBlockSlotLock.RUnlock()
 	defer s.latestCommitmentLock.RUnlock()
 	defer s.latestFinalizedSlotLock.RUnlock()
-	defer s.lastPrunedSlotLock.RUnlock()
+	defer s.lastPrunedEpochLock.RUnlock()
 
 	return &syncmanager.SyncStatus{
 		NodeSynced:             s.isBootstrappedFunc(),
 		LastAcceptedBlockSlot:  s.lastAcceptedBlockSlot,
 		LastConfirmedBlockSlot: s.lastConfirmedBlockSlot,
 		LatestCommitment:       s.latestCommitment,
 		LatestFinalizedSlot:    s.latestFinalizedSlot,
-		LatestPrunedSlot:       s.lastPrunedSlot,
+		LatestPrunedEpoch:      s.lastPrunedEpoch,
```

Comment:
heh we should align and use everywhere `latest`

Reply:
I prefer `Last`.  :grin:

Reply:
yeah, it's shorter but isn't last the actual "last" means: no further actions, and "latest": the most up to date?